cumulus/tasks/parse-pdr/tests/parse_pdrs_test.js

```diff
@@ -170,10 +157,7 @@ test('parse PDR from SFTP endpoint', async (t) => {
   }
   finally {
     // Clean up
-    await Promise.all([
-      recursivelyDeleteS3Bucket(internalBucketName),
-      fs.remove(providerPathDirectory)
-    ]);
+    await recursivelyDeleteS3Bucket(internalBucketName);
```

Comment:
I think this could be DRY'ed with a `test.after.always` block with line 110 and we wouldn't need these finally blocks